parameterspace/parameters/continuous.py

```diff
@@ -2,6 +2,7 @@
 # NOTICE file and/or the repository https://github.com/boschresearch/parameterspace
 #
 # SPDX-License-Identifier: Apache-2.0
+from cmath import isfinite
```

Comment:
Is there a reason that this is imported? Looks like the code below only uses `np.isfinite`.

Reply:
Apologies, I should have payed closer attention to the diff before submitting it. This is for sure just my IDE trying to be smart with import suggestions. I'll remove it, thanks :relaxed: